client/components/overviewWidget/AddToCart.jsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import axios from 'axios';
-import Sizes from './Sizes.jsx';
-import Quantities from './Quantities.jsx';
+import './styles/OverviewStyles.scss';
```

Comment:
@rhunce I didn't know you could import the .scss sheets directly into the component but it looks like it's working and not interfering with other styles. I would say that it's best to import OverviewStyles.scss into the main styles.scss so that Webpack compiles it with everyone else's sheets. I think it's fine to tackle that in a later PR though.